packages/date-picker/src/FilterDateRangePicker/components/DateRangeValidationMessage/DateRangeValidationMessage.spec.tsx

```diff
@@ -22,7 +22,8 @@ describe("<DateRangeValidationMessage />", () => {
     it("combines multiple validation messages with the same status", () => {
       const { container } = render(
         <DateRangeValidationMessage
-          id="error-message-id"
+          dateStartId="date-start-error-message-id"
```

Comment:
I see you adding these props to the render function, but they don't seem to be used in the spec.
Are those just mandatory props?

Reply:
They are not mandatory but I have added a test to confirm the ids are being passed down